ckcp/openshift_dev_setup.sh

```diff
@@ -277,7 +277,7 @@ patches:
       kcp_image_tag="$(yq -e '.images[0].newTag' "$ckcp_dev_dir"/kustomization.yaml)"
       KUBECONFIG="$KUBECONFIG_KCP" kubectl kcp workload sync local \
       --syncer-image ghcr.io/kcp-dev/kcp/syncer:$kcp_image_tag \
-      --resources conditions.tekton.dev,pipelines.tekton.dev,pipelineruns.tekton.dev,pipelineresources.tekton.dev,runs.tekton.dev,tasks.tekton.dev,taskruns.tekton.dev,repositories.pipelinesascode.tekton.dev > "$kube_dir/syncer.yaml"
+      --resources conditions.tekton.dev,pipelines.tekton.dev,pipelineruns.tekton.dev,pipelineresources.tekton.dev,runs.tekton.dev,repositories.pipelinesascode.tekton.dev > "$kube_dir/syncer.yaml"
```

Comment:
We should continue to sync Tasks, just not TaskRuns.

Syncing `Run` is something we should consider in a separate PR. This is used by Tekton Custom Tasks, which are not enabled by default.

```suggestion
      --resources conditions.tekton.dev,pipelines.tekton.dev,pipelineruns.tekton.dev,pipelineresources.tekton.dev,tasks.tekton.dev,runs.tekton.dev,repositories.pipelinesascode.tekton.dev > "$kube_dir/syncer.yaml"
```